lib/ui/home_page/components/home_view.dart

```diff
@@ -1,14 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get/get.dart' as GetX;
```

Comment:
I see this is a little odd in the code - sometimes get is imported as GetX, sometimes now. Also prior to this change, so this is an older issue.

Reply:
Thank you for pointing that out. I see the inconsistency with the import aliasing.If you have a preferred convention, like using **GetX** or keeping it as **get**, let me know, and I'll apply it across the code.